addons/website_popup/models/mass_mailing.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from openerp import api, fields, models
+
+
+class MassMailingList(models.Model):
+    _inherit = 'mail.mass_mailing.list'
+
+    popup_content = fields.Html(string="Popup content", translate=True, required=True, sanitize=False, default=lambda self: self._default_content())
+    popup_redirect_url = fields.Char(string="Popup Redirect URL")
+
+    @api.model
+    def _default_content(self):
+        return """<div class="text-center">
+                    <h2>Grow Your Business With Odoo Apps</h2>
+                    <h3>Join Odoo newsletter &amp; get answer instantly.</h3>
+                    <h3>Where Should We Send Your</h3>
+                    <h1>FREE</h1>
+                    Guide with sample Theme
+                    <h1> Odoo Design Templates</h1>
+                  </div>"""
```

Comment:
Hello @mba-odoo ,

We should move this banner text in data file and use it as default value for snippet and mailing list both because if i didn't select any newsletter while adding this snippet there will be no text on banner popup dialog.
See issue here : http://youtu.be/Wz5mLO8tO4A